tools/topology/topology2/cavs/cavs-nocodec.conf

```diff
@@ -33,16 +33,23 @@
 
 Define {
 	MCLK 				24576000
-	NUM_DMICS			0
 	# override DMIC default definitions
 	DMIC0_HOST_PIPELINE_ID		13
 	DMIC0_DAI_PIPELINE_ID		14
 	DMIC0_HOST_PIPELINE_SINK	'copier.host.13.1'
 	DMIC0_DAI_PIPELINE_SRC		'copier.DMIC.14.1'
+	DMIC1_HOST_PIPELINE_ID		15
+	DMIC1_DAI_PIPELINE_ID		16
+	DMIC1_HOST_PIPELINE_SINK	'copier.host.15.1'
+	DMIC1_DAI_PIPELINE_SRC		'copier.DMIC.16.1'
+	DMIC0_ID			6
+	DMIC1_ID			7
 	DMIC0_NAME			'NoCodec-6'
 	DMIC1_NAME			'NoCodec-7'
-	DMIC0_PCM_CAPS			'Passthrough Capture 13'
```

Comment:
How should the PCM text names be, what kind of names are preferred @plbossart ?

Reply:
@singalsu this is being updated and fixed by @jsarha - not blocking this merge though.